ginga/rv/plugins/Catalogs.py

```diff
@@ -67,13 +71,15 @@ def __init__(self, fv, fitsimage):
         canvas = self.dc.DrawingCanvas()
         canvas.enable_draw(True)
         canvas.set_drawtype(self.drawtype, color='cyan', linestyle='dash')
-        canvas.set_callback('cursor-down', self.btndown)
-        canvas.set_callback('cursor-up', self.btnup)
         canvas.set_callback('draw-event', self.draw_cb)
+        canvas.add_draw_mode('select', down=self.btndown,
+                             move=self.btndown, up=self.btnup)
```

Comment:
> Maybe it is best I leave that to you after this PR is merged

That sounds good, as it probably has not been well tested.  But basically it generalizes what has been done in the `btnup` callback.  There are some other interesting callbacks, like 'pick-hover', so you could do things like pop up a text box with the star name and information as you move the cursor over an object, etc.